Task3/Task3_Atanas_Kisyov/src/python/car_wash/main.py

```diff
@@ -0,0 +1,38 @@
+from db_connection.config import USER_CREDENTIALS
+from db_connection.connection_pools import ReadOnlyPool, ReadWritePool
+from utils.parse_arguments import add_arguments
+
+
+def main():
+    """Interact with user"""
+
+    #  Get arguments from user command
+    connection_options = add_arguments()
+    arg = {'arg': key for key, value in connection_options.items() if value}
```

Comment:
You do not need to do this

Reply:
How else should I get the passed argument? Or do you mean to move it in add_argument function?

Reply:
`if arg.ro: ...`
But Let's remove argparse and do our code easier

Reply:
sys.argv ?

Reply:
We will skip this functionality and spend time to fixe all other issues